NebulaWorld/SimulatedWorld.cs

```diff
@@ -67,26 +71,39 @@ public static void UpdateRemotePlayerAnimation(PlayerAnimationUpdate packet)
 
         public static void UpdatePlayerColor(ushort playerId, Float3 color)
         {
-            RemotePlayerModel player;
-            if (!remotePlayersModels.TryGetValue(playerId, out player))
+            Transform transform;
+            RemotePlayerModel remotePlayerModel;
+            if (LocalPlayer.IsMasterClient)
```

Comment:
This should be `playerId == LocalPlayer.PlayerId` instead of IsMasterClient, else you will apply the remote player color to the host.